siteapp/management/commands/screenshots.py

```diff
@@ -0,0 +1,266 @@
+        # Enable the mouse cursor following helper.
+        settings.MOUSE_CURSOR_FOLLOWER = True
+
+        # Get the organization that will be used.
+        self.org = Organization.objects.get(subdomain=options['org'])
+        self.subdomain = self.org.subdomain
+
+        # No database records we create here should live beyond
+        # this script. A database transaction would be handy, but
+        # the LiveServerTestCase uses a different database connection
+        # than what we have, and so it won't see the current transaction.
+        self.objects_to_kill = []
+
+        # Prepare for taking screenshots.
+        self.screenshot_basepath = options['path']
+        save_pdf = None
+        if options['path'].lower().endswith(".pdf"):
+            save_pdf = options['path']
+            self.screenshot_basepath += "-temp"
+        self.screenshot_image_filenames = []
+
+        # Run the script.
+        try:
+            if options['app']:
+                self.screenshot_app(options)
+
+            # Generate PDF.
+            if save_pdf:
+                # Write PDF.
+                from fpdf import FPDF
+                from PIL import Image
+                dpi = 120 # not that "pt" units are 1/72th of an inch
+                pdf = FPDF(unit="pt")
+                for image in self.screenshot_image_filenames:
+                    # Size the next PDF page to the size of this image.
+                    with open(image, "rb") as f:
+                        im = Image.open(f)
+                        page_size = im.size[0]/dpi*72, im.size[1]/dpi*72
+                    pdf.add_page(format=page_size)
+                    pdf.image(image, 0,0, page_size[0], page_size[1])
+                pdf.output(save_pdf, "F")
+
+                # Delete the temporary directory.
+                import shutil
+                shutil.rmtree(self.screenshot_basepath)
+
+        finally:
+            # Kill temporary objects.
+            for obj in reversed(self.objects_to_kill):
+                obj.delete()
+
+            # Terminate the browser.
+            self.browser.tearDown()
+            self.browser = None
+            SeleniumTest.tearDownClass()
+
+    def screenshot(self, slug, cursor_at=None):
+        if cursor_at:
+            elem = self.browser.browser.find_element_by_css_selector(cursor_at)
+            self.browser.browser.execute_script("moveMouseToElem(arguments[0]);", elem)
+            self.browser.browser.execute_script("arguments[0].scrollIntoView({ behavior: 'instant', block: 'nearest', inline: 'nearest' });", elem)
+            sleep(.1)
+
+        import os, os.path
+
+        os.makedirs(self.screenshot_basepath, exist_ok=True)
+
+        fn = os.path.join(
+            self.screenshot_basepath,
+            "{:03d}_{}.png".format(len(self.screenshot_image_filenames)+1, slug)
+            )
+        self.browser.browser.save_screenshot(fn)
+        self.screenshot_image_filenames.append(fn)
+
+    def login(self):
```

Comment:
Maybe we need to add explicit comments here about how we are addressing security? Or maybe this can only run when Debug is on?